src/client/Microsoft.Identity.Client/Http/Headers/AuthenticationInfoParameters.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Utils;
+using Microsoft.Identity.Json.Linq;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class AuthenticationInfoParameters
+    {
+        private const string _authenticationInfoKey = "Authentication-Info";
+        /// <summary>
+        /// 
+        /// </summary>
+        public string NextNonce { get; private set; }
```

Comment:
Should this also have "Error" and "ParserError" ?

Reply:
Yes, that makes sense